lib/image.mli

```diff
@@ -54,3 +54,13 @@ val get_portrait :
     - [None] if we don't have access to [p]'s portrait or it doesn't exist.
     - [Some src] with [src] the url or path of [p]'s portrait.
 *)
+
+val get_old_portrait :
+  config -> base -> person -> [> `Path of string | `Url of string ] option
+(** [get_portrait conf base p] is
+    - [None] if we don't have access to [p]'s portrait or it doesn't exist.
+    - [Some src] with [src] the url or path of [p]'s portrait.
+*)
+
+val get_keydir_files : config -> base -> person -> bool -> string list
```

Comment:
The last `bool` argument can be a little confusing and forces to read the documentation or the function code to understand.
If you think it is relevant you can expose two function in the mli `get_keydir_files` and `get_old_keydir_files`, that call the same `get_keydir_files_aux` function but handle the bool argument for us.

By the way, why do we have a "old" folder? Is it to archive images?